pytorch_lightning/profiler/pytorch.py

```diff
@@ -198,12 +272,33 @@ def __init__(
         self._register: Optional[RegisterRecordFunction] = None
         self._parent_profiler: Optional[_PROFILER] = None
         self._recording_map: Dict[str, record_function] = {}
-
-        if self._export_to_chrome and self._path_to_export_trace is None:
-            rank_zero_warn(
-                "The exported trace would be saved locally as `path_to_export_trace` is None."
-                " Note: Each functions will generate its own traced file."
-            )
+        self._start_action_name: Optional[str] = None
+        self._schedule: Optional[ScheduleWrapper] = None
+
+        if _KINETO_AVAILABLE:
+            has_schedule = "schedule" in profiler_kwargs
+            self._has_on_trace_ready = "on_trace_ready" in profiler_kwargs
+
+            schedule = profiler_kwargs.get("schedule", None)
+            if schedule is not None:
+                if not isinstance(schedule, Callable):
+                    raise MisconfigurationException(f"Schedule should be a callable. Found: {schedule}")
+                action = schedule(0)
```

Comment:
won't this update the scheduler steps?

Reply:
No, schedule in just a function. Doesn't have any state.